packages/components/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 -   `Draggable`: Convert to TypeScript ([#45471](https://github.com/WordPress/gutenberg/pull/45471)).
 -   `MenuGroup`: Convert to TypeScript ([#45617](https://github.com/WordPress/gutenberg/pull/45617)).
 -   `useCx`: fix story to satisfy the `react-hooks/exhaustive-deps` eslint rule ([#45614](https://github.com/WordPress/gutenberg/pull/45614))
+-   `AnglePickerControl`: remove `:focus-visible` outline on `CircleOutlineWrapper` ([#45758](https://github.com/WordPress/gutenberg/pull/45758))
```

Comment:
We'll likely need to move this entry to the most recent `Unreleased` section, since there has been a package release in the meantime